docs/getting-started-ei.md

```diff
@@ -0,0 +1,97 @@
+# Getting Started - WSO2 Enterprise Integrator
+
+Setting up a basic pipeline for WSO2 Enterprise Integrator on Kubernetes is quick and simple.
+
+You can set up a simple CI/CD pipeline for WSO2 Enterprise Integrator in two steps.
+- Create a Docker image for WSO2 Enterprise Integrator.
+- Deploy the CI/CD pipeline.
+
+Before you begin to develop your pipeline, set up the following prerequisites in your Kubernetes cluster.
+
+## Prerequisites
+
+- Install and set up Helm.
+- Install Nginx Ingress Controller Git release nginx-0.22.0.
+
+
+## Create WSO2 Enterprise Integrator Docker Image
+
+The first step in developing the pipeline is to create an image for WSO2 Enterprise Integrator on a private Docker registry. This pipeline utilizes Docker Hub as its private registry.
+
+After creating the repository, pull the latest image from WSO2 and push it to the private registry.
+
+- If you have a WSO2 subscription, log into the WSO2 Docker registry using your subscription credentials.
+
+    ```
+    docker login docker.wso2.com
+    docker pull docker.wso2.com/wso2ei-integrator:6.5.0
+    docker tag docker.wso2.com/wso2ei-integrator:6.5.0 <DOCKER_ORGANIZATION>/wso2ei-integrator
+    ```
+
+    ><DOCKER_ORGANIZATION> refers to the name of your organization in Docker Hub.
+
+- If you do not have a WSO2 subscription use the docker hub image wso2/wso2ei-integrator:6.5.0 instead.
+
+    ```
+    docker pull wso2/wso2ei-integrator:6.5.0
+    docker tag wso2/wso2ei-integrator:6.5.0<DOCKER_ORGANIZATION>/wso2ei-integrator
+    ```
+
+> The WSO2 Enterprise Integrator Docker image automatically fetches product updates on a weekly basis. If you do not have a WSO2 subscription account, you can sign up for a free trial here.
+
+Then, log into your organization on Docker Hub and push the Enterprise Integrator image.
+
+```
+docker login
+docker push <DOCKER_ORGANIZATION>/wso2ei-integrator
+```
+
+## Deploy the CI/CD pipeline
+
+1. Download the [sample values.yaml](../samples/values-ei-pattern-1.yaml) file and replace the placeholders with their respective values.
+
+    - <WSO2_SUBSCRIPTION_USERNAME>
+    - <WSO2_SUBSCRIPTION_PASSWORD>
+    - <REGISTRY_USERNAME>
+    - <REGISTRY_PASSWORD>
+    - <REGISTRY_EMAIL>
+    - \<EMAIL>
+    - <GITHUB_USERNAME>
+    - <GITHUB_PASSWORD>
+
+2. Add the WSO2 helm repository.
```

Comment:
@Aaquiff has to be `Helm`. This applies to documentation of all products.

Reply:
Fixed